### PR TITLE
Merge rel/1.0.0 into feature/msbuild

### PR DESCRIPTION
**Pull request by piotrpMSFT**
reversing the merge to make PR review easier

### CODE REVIEW

**Comment by eerhardt** [comment]
> // Copyright (c) .NET Foundation and contributors. All rights reserved.

This file can be removed since DependencyModel doesn't exist in feature/msbuild anymore.

---
Refers to: src/Microsoft.Extensions.DependencyModel/JsonTextReaderExtensions.cs:1 in 7993b43. [](commit_id = 7993b438a0cfc6250b83ff9d23215e402304244a, deletion_comment = False)

File: build/package/Microsoft.DotNet.Cli.Installer.DEB.props [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    <DotnetDebToolPackageId>dotnet-deb-tool</DotnetDebToolPackageId>  
+    <DotnetDebToolVersion Condition=" '$(DotnetDebToolVersion)' == '' ">1.0.0-*</DotnetDebToolVersion>
+    <DotnetDebToolConsumerProjectName>dotnet-deb-tool-consumer</DotnetDebToolConsumerProjectName>
+    <DotnetDebToolDir>$(IntermediateDirectory)/$(DotnetDebToolConsumerProjectName)</DotnetDebToolDir>
+    <DotnetDebToolPackageSource Condition=" '$(DotnetDebToolPackageSource)' == '' "></DotnetDebToolPackageSource>
+  </PropertyGroup>  
+
+  <ItemGroup>
+    <DotnetDebToolProjectJsonLines Include="{" />
+    <DotnetDebToolProjectJsonLines Include="%20%20%20%20&quot;version&quot;: &quot;1.0.0-*&quot;," />
```
Comment (eerhardt) [inline build/package/Microsoft.DotNet.Cli.Installer.DEB.props]: This isn't really readable.  How about we put this in a .json file and then read that file here?

**Comment by eerhardt** [comment]
:shipit:

**Comment by piotrpMSFT** [comment]
@dotnet-bot test Windows_NT x86 Debug Build please.

**Comment by livarcocc** [comment]
:shipit:

**Pull request merged**